src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesNumber = Math.ceil(total / perPage);
+  const pages = getNumbers(1, pagesNumber);
+  const ifFirst = () => currentPage === 1;
```

Comment:
Instead of creating new functions inside the component for checking currentPage, you can just do the comparisons inline - it's enough in this case

Reply:
I created a new function, because i need to check currentPage 3 times. To better understanding and if in future i wll need to change the logic of my code, its better to use function

Reply:
it is ok for `ifCur` because it's taken an argument. But why don't do just
`const ifFirst = currentPage === 1;`?